VKApp/VKApp/Extensions/Alert+Extension.swift

```diff
@@ -0,0 +1,21 @@
+// Alert+Extension.swift
```

Comment:
```suggestion
// UIViewController+Alert.swift
```
Поменяй название файла